cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -37,7 +42,7 @@ func init() {
 	klog.InitFlags(klogFlags)
 
 	// Sync the glog and klog flags.
-	flag.CommandLine.VisitAll(func(f1 *flag.Flag) {
+	flag.CommandLine.Visit(func(f1 *flag.Flag) {
```

Comment:
this addresses an issue that was already fixed in the openshift fork of this csi driver, but was never pushed upstream here. Without this fix, the args fail to parse.